zerver/openapi/zulip.yaml

```diff
@@ -2247,15 +2247,15 @@ paths:
                                     The choice the editing user made about which messages should be
                                     affected by a stream/topic edit:
 
-                                    - `change_one` => Just change the one indicated in `message_id`.
-                                    - `change_later` => Change messages in the same topic that had
+                                    - `"change_one"`: Just change the one indicated in `message_id`.
+                                    - `"change_later"`: Change messages in the same topic that had
```

Comment:
We could make it a rule to not have "=>" in `zulip.yaml`, but I wonder if it will happen now that I've removed all the legacy cases?

Reply:
Yeah, probably not; I'm pretty sure I added all the ones that exist myself.